src/test/regress/expected/foreign_key_restriction_enforcement.out

```diff
@@ -722,7 +722,7 @@ BEGIN;
 
 	UPDATE reference_table SET id = 101 WHERE id = 99;
 ERROR:  cannot modify table "reference_table" because there was a parallel operation on a distributed table
-DETAIL: When there is a foreign key to a reference table or to a local table, Citus needs to perform all operations over a single connection per node to ensure consistency.
+DETAIL:  When there is a foreign key to a reference table or to a local table, Citus needs to perform all operations over a single connection per node to ensure consistency.
```

Comment:
does this need a normalize_expected.sh?

Reply:
This change is triggered by normalize_expected. As MacOS has some minor differences, I checked with Ozan on ubuntu, and ci/normalize_expected.sh produced no diff on top of this.